### ISSUE TITLE
[SUGGESTION] Add Support For Slime World Manager

### ISSUE THREAD

**Issue by Ant4rez** [IridiumLLC/IridiumSkyblock#36]
**Describe the Suggestion**
This plugin makes worlds load faster, use less space disk, and even load the world through a database. I think it can be a very good feature add support for this on the plugin.

GitHub: https://github.com/Grinderwolf/Slime-World-Manager

Thanks:).

**Comment by IridiumLLC** [comment]
"Note that SWM is not intended to be used on big survival worlds, but small worlds like minigame maps or server lobbies. There's more on this on the FAQ section." Wouldnt the skyblock world count as a big survival world as it extends indefinitely for when people do /is create?

**Comment by Ant4rez** [comment]
> "Note that SWM is not intended to be used on big survival worlds, but small worlds like minigame maps or server lobbies. There's more on this on the FAQ section." Wouldnt the skyblock world count as a big survival world as it extends indefinitely for when people do /is create?

You are right, but there is no clarity on how bigger can be the map to doesn't work fine. And the only way that the "skyworld" can be a big survival map is having a loot of players in the same world and for small servers i thinks this feature is good, also this can be complemented with other features, like having a world per player like this plugin (https://www.spigotmc.org/resources/worldsystem-●-the-one-world-per-player-solution-●-gui-●-highly-configurable-●-1-8-1-14.49756/) , but each world generated with SWM is for one player only or his party. Even you can add a bungeecord feature where 1 server can be the island generated with SWM for the same purpoeses this can be very good for big servers.

**Comment by JDChris100** [comment]
Just to keep you updated, Im still questioning if i should do this. I probabbly will, but its kinda low on my priorities list. and it will be a configurable options. and if it is enabled players will have per world islands

**Comment by IridiumLLC** [comment]
Just to keep you updated, I probably will add this, however it is low in my list.

**Comment by Ant4rez** [comment]
> Just to keep you updated, I probably will add this, however it is low in my list.

Thanks for answer, i appreciate it :).

**Comment by JustinBery** [comment]
Hello, I'd like to ask why the player generated the island
There will be land
And the land will not be cleared after expanding and upgrading the air Island, but can be used by players

**Comment by IridiumLLC** [comment]
Not sure what you mean, but if you mean that normal land is being genearted instead of a void this was fixed in the latest version. But please either create your own bug report or join my discord :)

**Comment by JustinBery** [comment]
After players receive the empty Island
There will be continents.
Upgrade size will cover mainland

I am Chinese
I'm sorry for my poor English

**Comment by JustinBery** [comment]
Around the island, there is the original land map

**Comment by JustinBery** [comment]
It has flat terrain near the island, why?
